tests/transform_dialect/matmul_fill_spec_pad_pack.mlir

```diff
@@ -135,3 +149,39 @@ module attributes { transform.with_named_sequence } {
     transform.yield
   }
 }
+
+// CHECK-LABEL: @matmul_i32
+//       CHECK: scf.forall
+//       CHECK: {
+//       CHECK:   memref.alloc() : memref<64x2048xi32, 1>
+//       CHECK:   scf.for
+//       CHECK:   {
+//       CHECK:     linalg.copy ins(%{{.*}} : memref<64x256xi32, strided<[2048, 1], offset: ?>, #hal.descriptor_type<storage_buffer>>) outs(%{{.*}} : memref<64x256xi32, strided<[2048, 1], offset: ?>, 1>)
+//       CHECK:   }
+//       CHECK:   memref.alloc() : memref<2048x64xi32, 1>
+//       CHECK:   scf.for
+//       CHECK:   {
+//       CHECK:     linalg.copy ins(%{{.*}} : memref<256x64xi32, strided<[512, 1], offset: ?>, #hal.descriptor_type<storage_buffer>>) outs(%{{.*}} : memref<256x64xi32, strided<[64, 1], offset: ?>, 1>)
+//       CHECK:   }
+//       CHECK:   memref.alloc() : memref<64x64xi32, 1>
+//       CHECK:   scf.forall
+//       CHECK:   {
+//       CHECK:     linalg.fill ins(%{{.*}}) outs(%{{.*}} : memref<4x8x4x8xi32, 2>)
+//       CHECK:     scf.for
+//       CHECK:     {
+//       CHECK:       memref.alloc() : memref<4x8x4x8xi32, 2>
+//       CHECK:       iree_linalg_ext.pack %{{.*}} outer_dims_perm = [1, 0] inner_dims_pos = [0, 1] inner_tiles = [4, 8] into %{{.*}} : (memref<32x32xi32, strided<[2048, 1], offset: ?>, 1> memref<4x8x4x8xi32, 2>)
+//       CHECK:       memref.alloc() : memref<4x4x8x8xi32, 2>
+//       CHECK:       iree_linalg_ext.pack %{{.*}} outer_dims_perm = [1, 0] inner_dims_pos = [0, 1] inner_tiles = [8, 8] into %{{.*}} : (memref<32x32xi32, strided<[64, 1], offset: ?>, 1> memref<4x4x8x8xi32, 2>)
+//       CHECK:       linalg.generic
+//       CHECK:       memref.dealloc %{{.*}} : memref<4x8x4x8xi32, 2>
+//       CHECK:       memref.dealloc %{{.*}} : memref<4x4x8x8xi32, 2>
+//       CHECK:     }
+//       CHECK:     iree_linalg_ext.unpack %{{.*}} outer_dims_perm = [1, 0] inner_dims_pos = [0, 1] inner_tiles = [4, 8] into %{{.*}} : (memref<4x8x4x8xi32, 2> memref<32x32xi32, strided<[64, 1], offset: ?>, 1>)
+//       CHECK:     memref.dealloc %{{.*}} : memref<4x8x4x8xi32, 2>
+//       CHECK:   }
+//       CHECK:   linalg.copy ins(%{{.*}} : memref<64x64xi32, 1>) outs(%{{.*}} : memref<64x64xi32, strided<[512, 1], offset: ?>, #hal.descriptor_type<storage_buffer>>)
+//       CHECK:   memref.dealloc %{{.*}} : memref<64x2048xi32, 1>
+//       CHECK:   memref.dealloc %{{.*}} : memref<2048x64xi32, 1>
```

Comment:
Maybe remove checks for dealloc, I'm assuming they're not important to the transform being tested

Reply:
I would keep the check for `memref.dealloc`. The location of these alloc and dealloc ops are important in AIR, so want to make it clear when someone look at this test IR.